docker-compose.yml

```diff
@@ -73,6 +86,8 @@ services:
     working_dir: /pixel
     env_file:
       - .env
+    environment:
+      - DISPLAY=novnc:0
```

Comment:
As a reminder, this line makes the magic happen

It causes the visual-regression container to use the novnc container _as its display_

The Backstop process (including Chrome) still runs in the visual-regression container, but the display window for Chrome appears on the novnc desktop

We then can see the novnc container's desktop because the novnc program lets make a vnc connections from a browser, in this case from the host 

The novnc container's desktop is using xfce4 atop an xvfb virtual frame buffer

websockify lets the host connect to the novnc container's novnc

Reply:
Here's a subtle point I needed clarifying on even after getting this all working:

![Screenshot 2024-07-08 at 1 10 39 PM](https://github.com/wikimedia/pixel/assets/3143487/be0f9689-a42e-4e3c-8f13-66d5d27d36eb)